egs/wsj/s5/utils/format_lm_sri.sh

```diff
@@ -104,6 +104,27 @@ awk '{print $1}' $out_dir/words.txt > $tmpdir/voc || exit 1;
 change-lm-vocab -vocab $tmpdir/voc -lm $tmpdir/lm.gz -write-lm $tmpdir/out_lm \
   $srilm_opts || exit 1;
 
+# change-lm-vocab introduces error in out_lm when lm.gz has an order
```

Comment:
@mmmaat: I think I understand where and why this happens, but it would be very helpful if you either attached the failing ARPA file (if it's not very large) to the ticket I opened to this issue, #643, or just posted the directives and ngram counts from the file, as below:
```
$ zcat lm_tgsmall.arpa.gz | grep '^\\\|^ngram'
\data\
ngram 1=200003
ngram 2=1016673
ngram 3=340026
\1-grams:
\2-grams:
\3-grams:
\end\
```